cupy/_creation/from_data.py

```diff
@@ -28,6 +29,10 @@ def array(obj, dtype=None, copy=True, order='K', subok=False, ndmin=0):
             array (default).
         ndmin (int): Minimum number of dimensions. Ones are inserted to the
             head of the shape if needed.
+        blocking (bool): Default is ``False``, meaning if a H2D copy is needed
+            it would run asynchronously on the current stream, and users are
+            responsible for ensuring the stream order. If set to ``True``,
+            the copy is synchronous (with respect to the host).
```

Comment:
How about documenting that users would need to set it to True if the source object is might be modified concurrently during copy?

Reply:
See commit 2810dbc.